chanacceptor/merge.go

```diff
@@ -0,0 +1,152 @@
+package chanacceptor
+
+import (
+	"bytes"
+	"fmt"
+
+	"github.com/btcsuite/btcutil"
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+const (
+	// We use field names in our errors for more readable errors. Create
+	// consts for them here so that we can exactly match in our unit tests.
+	fieldCSV             = "csv delay"
+	fieldHtlcLimit       = "htlc limit"
+	fieldMinDep          = "min depth"
+	fieldReserve         = "reserve"
+	fieldMinIn           = "min htlc in"
+	fieldInFlightTotal   = "in flight total"
+	fieldUpfrontShutdown = "upfront shutdown"
+)
+
+// mergeError returns a merge error for a named field when we get two channel
+// acceptor responses which have different values set.
+func mergeError(name string, current, new interface{}) error {
```

Comment:
Confusingly named because it doesnt merge two `error` types

Reply:
renamed!